src/main/java/com/gtnewhorizons/retrofuturabootstrap/api/FastClassAccessor.java

```diff
@@ -31,6 +31,8 @@ public interface FastClassAccessor {
     @Nullable
     String binarySuperName();
 
+    String @NotNull [] binaryInterfaceNames();
```

Comment:
The accessor should return an unmodifiable `@NotNull List<@NotNull String>` that can be cached in the implementation, that should avoid reallocating a new list on every invocation